### ISSUE TITLE
Migrate from heroku

### ISSUE THREAD

**Issue by cdock1029** [superfly/flyctl#118]
I've followed the steps here:

https://fly.io/docs/app-guides/speed-up-a-heroku-app/#getting-started-with-turboku

Presumably this has created the necessary Dockerfile behind the scenes.
What is the next step to "move off Heroku" as advertised? I'm trying to "deploy an image" from my local repo, and of course getting:

```
==> Validating app configuration
--> done
Error Project does not contain a Dockerfile or specify a builder
```

The transition from Heroku seems incomplete. How do I access the project files setup for me? Push new code?

**Comment by michaeldwan** [comment]
@cdock1029 each Heroku release updates your fly app, so you'd continue doing what you've been doing and your fly app will stay updated. If you want to migrate the app entirely to fly you would need to create a Dockerfile (or use available buildpacks) so flyctl can create standalone images to deploy to fly.

We don't current have a guide for migrating from heroku to fly after using the turboku button, but we can put something together. /cc @codepope

**Comment by fu-sen** [comment]
I have tried this migration with PHP. Other languages may be helpful:
https://github.com/fu-sen/fly-php-heroku-example

**Comment by codepope** [comment]
A note has been added to the app guide on the linkage between the Heroku app and the Fly/Turboku app.

**Comment by 4mirai** [comment]
It doesn't seem to auto detect deployments to heroku and the fly app doesn't get updated.

**Comment by michaeldwan** [comment]
We don't update your app on deploys to Heroku anymore. It was hard to reliably handle all the various Heroku configurations or problems that could happen, so we decided it was better for our tool to help you test your app on Fly.io then get out of the way. If you like it, you can deploy directly to Fly.io along with or instead of Heroku.

**Comment by JoeGermuska** [comment]
Is there a way to extract the fly.toml file that was created with Turboku?

Before I tried Turboku, I wrestled with creating one for my simple Flask/SQLite app. 

With buildpacks, I couldn't find a setting that had proper SQLite support.

With a Dockerfile, even though the Dockerfile built an image and ran perfectly locally, when I tried `fly deploy` it just hung for almost an hour before I remembered I was waiting for it and killed it.
